lib/smack/SmackRep.cpp

```diff
@@ -154,8 +210,8 @@ void SmackRep::collectRegions(llvm::Module &M) {
 
 const Stmt* SmackRep::alloca(llvm::AllocaInst& i) {  
   const Expr* size = 
-    Expr::fn("$mul",lit(storageSize(i.getAllocatedType())),lit(i.getArraySize()));
-                       
+    Expr::fn(opName("$mul", {TRUNC(getIntSize(i.getOperand(0)))}), lit(storageSize(i.getAllocatedType())),lit(i.getArraySize()));
```

Comment:
Yes, it is. Maybe I can try 64 bit pointer and see how the verification time goes. Would it be intuitive that 32 bit pointer takes less time?